23/06/17 10:16:17 INFO client.RMProxy: Connecting to ResourceManager at /127.0.0.1:8032
23/06/17 10:16:18 WARN mapreduce.JobResourceUploader: Hadoop command-line option parsing not performed. Implement the Tool interface and execute your application with ToolRunner to remedy this.
23/06/17 10:16:20 INFO input.FileInputFormat: Total input paths to process : 16
23/06/17 10:16:20 INFO mapreduce.JobSubmitter: number of splits:16
23/06/17 10:16:21 INFO mapreduce.JobSubmitter: Submitting tokens for job: job_1686987592378_0004
23/06/17 10:16:23 INFO impl.YarnClientImpl: Submitted application application_1686987592378_0004
23/06/17 10:16:23 INFO mapreduce.Job: The url to track the job: http://pcvm603-12.emulab.net:8088/proxy/application_1686987592378_0004/
23/06/17 10:16:23 INFO mapreduce.Job: Running job: job_1686987592378_0004
23/06/17 10:16:53 INFO mapreduce.Job: Job job_1686987592378_0004 running in uber mode : false
23/06/17 10:16:53 INFO mapreduce.Job:  map 0% reduce 0%
23/06/17 10:22:39 INFO ipc.Client: Retrying connect to server: pcvm604-13.emulab.net/172.20.4.13:45373. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=3, sleepTime=1000 MILLISECONDS)
23/06/17 10:22:40 INFO ipc.Client: Retrying connect to server: pcvm604-13.emulab.net/172.20.4.13:45373. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=3, sleepTime=1000 MILLISECONDS)
23/06/17 10:22:41 INFO ipc.Client: Retrying connect to server: pcvm604-13.emulab.net/172.20.4.13:45373. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=3, sleepTime=1000 MILLISECONDS)
23/06/17 10:30:59 INFO mapreduce.Job:  map 4% reduce 0%
23/06/17 10:31:15 INFO mapreduce.Job:  map 15% reduce 0%
23/06/17 10:31:19 INFO mapreduce.Job:  map 21% reduce 0%
23/06/17 10:31:20 INFO mapreduce.Job:  map 23% reduce 0%
23/06/17 10:31:24 INFO mapreduce.Job:  map 27% reduce 0%
23/06/17 10:31:33 INFO mapreduce.Job:  map 29% reduce 0%
23/06/17 10:31:34 INFO mapreduce.Job:  map 33% reduce 0%
23/06/17 10:31:36 INFO mapreduce.Job:  map 38% reduce 0%
23/06/17 10:32:37 INFO mapreduce.Job:  map 38% reduce 13%
23/06/17 10:33:29 INFO mapreduce.Job:  map 44% reduce 13%
23/06/17 10:34:14 INFO mapreduce.Job:  map 44% reduce 15%
23/06/17 10:34:20 INFO mapreduce.Job:  map 48% reduce 15%
23/06/17 10:34:52 INFO mapreduce.Job:  map 94% reduce 15%
23/06/17 10:35:08 INFO mapreduce.Job:  map 100% reduce 15%
23/06/17 10:35:14 INFO mapreduce.Job:  map 100% reduce 19%
23/06/17 10:35:16 INFO mapreduce.Job:  map 100% reduce 21%
23/06/17 10:35:52 INFO mapreduce.Job:  map 100% reduce 25%
23/06/17 10:36:01 INFO mapreduce.Job: Task Id : attempt_1686987592378_0004_m_000013_1000, Status : FAILED
AttemptID:attempt_1686987592378_0004_m_000013_1000 Timed out after 600 secs
cleanup failed for container container_1686987592378_0004_02_000015 : java.io.IOException: Failed on local exception: java.io.IOException: java.net.SocketTimeoutException: 60000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/172.20.5.7:39018 remote=pcvm604-13.emulab.net/172.20.4.13:45179]; Host Details : local host is: "slave1.hadoop-fair.mapreduce630.emulab.net/172.20.5.7"; destination host is: "pcvm604-13.emulab.net":45179; 
	at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:776)
	at org.apache.hadoop.ipc.Client.call(Client.java:1480)
	at org.apache.hadoop.ipc.Client.call(Client.java:1413)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Invoker.invoke(ProtobufRpcEngine.java:229)
	at com.sun.proxy.$Proxy81.stopContainers(Unknown Source)
	at org.apache.hadoop.yarn.api.impl.pb.client.ContainerManagementProtocolPBClientImpl.stopContainers(ContainerManagementProtocolPBClientImpl.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:191)
	at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:102)
	at com.sun.proxy.$Proxy82.stopContainers(Unknown Source)
	at org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl$Container.kill(ContainerLauncherImpl.java:207)
	at org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl$EventProcessor.run(ContainerLauncherImpl.java:379)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: java.net.SocketTimeoutException: 60000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/172.20.5.7:39018 remote=pcvm604-13.emulab.net/172.20.4.13:45179]
	at org.apache.hadoop.ipc.Client$Connection$1.run(Client.java:688)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1758)
	at org.apache.hadoop.ipc.Client$Connection.handleSaslConnectionFailure(Client.java:651)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:738)
	at org.apache.hadoop.ipc.Client$Connection.access$2900(Client.java:376)
	at org.apache.hadoop.ipc.Client.getConnection(Client.java:1529)
	at org.apache.hadoop.ipc.Client.call(Client.java:1452)
	... 16 more
Caused by: java.net.SocketTimeoutException: 60000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/172.20.5.7:39018 remote=pcvm604-13.emulab.net/172.20.4.13:45179]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:161)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:131)
	at java.io.FilterInputStream.read(FilterInputStream.java:133)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at org.apache.hadoop.security.SaslRpcClient.saslConnect(SaslRpcClient.java:367)
	at org.apache.hadoop.ipc.Client$Connection.setupSaslConnection(Client.java:561)
	at org.apache.hadoop.ipc.Client$Connection.access$1900(Client.java:376)
	at org.apache.hadoop.ipc.Client$Connection$2.run(Client.java:730)
	at org.apache.hadoop.ipc.Client$Connection$2.run(Client.java:726)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1758)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:726)
	... 19 more

23/06/17 10:36:01 INFO mapreduce.Job: Task Id : attempt_1686987592378_0004_m_000010_1000, Status : FAILED
AttemptID:attempt_1686987592378_0004_m_000010_1000 Timed out after 600 secs
cleanup failed for container container_1686987592378_0004_02_000012 : java.io.IOException: Failed on local exception: java.io.IOException: java.net.SocketTimeoutException: 60000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/172.20.5.7:39016 remote=pcvm604-13.emulab.net/172.20.4.13:45179]; Host Details : local host is: "slave1.hadoop-fair.mapreduce630.emulab.net/172.20.5.7"; destination host is: "pcvm604-13.emulab.net":45179; 
	at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:776)
	at org.apache.hadoop.ipc.Client.call(Client.java:1480)
	at org.apache.hadoop.ipc.Client.call(Client.java:1413)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Invoker.invoke(ProtobufRpcEngine.java:229)
	at com.sun.proxy.$Proxy81.stopContainers(Unknown Source)
	at org.apache.hadoop.yarn.api.impl.pb.client.ContainerManagementProtocolPBClientImpl.stopContainers(ContainerManagementProtocolPBClientImpl.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:191)
	at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:102)
	at com.sun.proxy.$Proxy82.stopContainers(Unknown Source)
	at org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl$Container.kill(ContainerLauncherImpl.java:207)
	at org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl$EventProcessor.run(ContainerLauncherImpl.java:379)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: java.net.SocketTimeoutException: 60000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/172.20.5.7:39016 remote=pcvm604-13.emulab.net/172.20.4.13:45179]
	at org.apache.hadoop.ipc.Client$Connection$1.run(Client.java:688)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1758)
	at org.apache.hadoop.ipc.Client$Connection.handleSaslConnectionFailure(Client.java:651)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:738)
	at org.apache.hadoop.ipc.Client$Connection.access$2900(Client.java:376)
	at org.apache.hadoop.ipc.Client.getConnection(Client.java:1529)
	at org.apache.hadoop.ipc.Client.call(Client.java:1452)
	... 16 more
Caused by: java.net.SocketTimeoutException: 60000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/172.20.5.7:39016 remote=pcvm604-13.emulab.net/172.20.4.13:45179]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:161)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:131)
	at java.io.FilterInputStream.read(FilterInputStream.java:133)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at org.apache.hadoop.security.SaslRpcClient.saslConnect(SaslRpcClient.java:367)
	at org.apache.hadoop.ipc.Client$Connection.setupSaslConnection(Client.java:561)
	at org.apache.hadoop.ipc.Client$Connection.access$1900(Client.java:376)
	at org.apache.hadoop.ipc.Client$Connection$2.run(Client.java:730)
	at org.apache.hadoop.ipc.Client$Connection$2.run(Client.java:726)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1758)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:726)
	... 19 more

23/06/17 10:36:01 INFO mapreduce.Job: Task Id : attempt_1686987592378_0004_m_000011_1000, Status : FAILED
AttemptID:attempt_1686987592378_0004_m_000011_1000 Timed out after 600 secs
cleanup failed for container container_1686987592378_0004_02_000013 : java.io.IOException: Failed on local exception: java.io.IOException: java.net.SocketTimeoutException: 60000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/172.20.5.7:39020 remote=pcvm604-13.emulab.net/172.20.4.13:45179]; Host Details : local host is: "slave1.hadoop-fair.mapreduce630.emulab.net/172.20.5.7"; destination host is: "pcvm604-13.emulab.net":45179; 
	at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:776)
	at org.apache.hadoop.ipc.Client.call(Client.java:1480)
	at org.apache.hadoop.ipc.Client.call(Client.java:1413)
	at org.apache.hadoop.ipc.ProtobufRpcEngine$Invoker.invoke(ProtobufRpcEngine.java:229)
	at com.sun.proxy.$Proxy81.stopContainers(Unknown Source)
	at org.apache.hadoop.yarn.api.impl.pb.client.ContainerManagementProtocolPBClientImpl.stopContainers(ContainerManagementProtocolPBClientImpl.java:110)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:191)
	at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:102)
	at com.sun.proxy.$Proxy82.stopContainers(Unknown Source)
	at org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl$Container.kill(ContainerLauncherImpl.java:207)
	at org.apache.hadoop.mapreduce.v2.app.launcher.ContainerLauncherImpl$EventProcessor.run(ContainerLauncherImpl.java:379)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: java.net.SocketTimeoutException: 60000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/172.20.5.7:39020 remote=pcvm604-13.emulab.net/172.20.4.13:45179]
	at org.apache.hadoop.ipc.Client$Connection$1.run(Client.java:688)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1758)
	at org.apache.hadoop.ipc.Client$Connection.handleSaslConnectionFailure(Client.java:651)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:738)
	at org.apache.hadoop.ipc.Client$Connection.access$2900(Client.java:376)
	at org.apache.hadoop.ipc.Client.getConnection(Client.java:1529)
	at org.apache.hadoop.ipc.Client.call(Client.java:1452)
	... 16 more
Caused by: java.net.SocketTimeoutException: 60000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/172.20.5.7:39020 remote=pcvm604-13.emulab.net/172.20.4.13:45179]
	at org.apache.hadoop.net.SocketIOWithTimeout.doIO(SocketIOWithTimeout.java:164)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:161)
	at org.apache.hadoop.net.SocketInputStream.read(SocketInputStream.java:131)
	at java.io.FilterInputStream.read(FilterInputStream.java:133)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at org.apache.hadoop.security.SaslRpcClient.saslConnect(SaslRpcClient.java:367)
	at org.apache.hadoop.ipc.Client$Connection.setupSaslConnection(Client.java:561)
	at org.apache.hadoop.ipc.Client$Connection.access$1900(Client.java:376)
	at org.apache.hadoop.ipc.Client$Connection$2.run(Client.java:730)
	at org.apache.hadoop.ipc.Client$Connection$2.run(Client.java:726)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1758)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:726)
	... 19 more

23/06/17 10:36:02 INFO mapreduce.Job:  map 81% reduce 27%
23/06/17 10:38:09 INFO mapreduce.Job:  map 94% reduce 27%
23/06/17 10:38:10 INFO mapreduce.Job:  map 100% reduce 27%
23/06/17 10:38:30 INFO mapreduce.Job:  map 100% reduce 33%
23/06/17 10:38:33 INFO mapreduce.Job:  map 100% reduce 45%
23/06/17 10:38:36 INFO mapreduce.Job:  map 100% reduce 67%
23/06/17 10:38:42 INFO mapreduce.Job:  map 100% reduce 100%
23/06/17 10:39:58 INFO mapreduce.Job: Job job_1686987592378_0004 completed successfully
23/06/17 10:40:00 INFO mapreduce.Job: Counters: 52
	File System Counters
		FILE: Number of bytes read=86558
		FILE: Number of bytes written=2256490
		FILE: Number of read operations=0
		FILE: Number of large read operations=0
		FILE: Number of write operations=0
		HDFS: Number of bytes read=127667
		HDFS: Number of bytes written=59112
		HDFS: Number of read operations=51
		HDFS: Number of large read operations=0
		HDFS: Number of write operations=2
	Job Counters 
		Failed map tasks=8
		Killed map tasks=3
		Launched map tasks=26
		Launched reduce tasks=1
		Other local map tasks=5
		Data-local map tasks=21
		Total time spent by all maps in occupied slots (ms)=9623706
		Total time spent by all reduces in occupied slots (ms)=491259
		Total time spent by all map tasks (ms)=9623706
		Total time spent by all reduce tasks (ms)=491259
		Total vcore-milliseconds taken by all map tasks=9623706
		Total vcore-milliseconds taken by all reduce tasks=491259
		Total megabyte-milliseconds taken by all map tasks=9854674944
		Total megabyte-milliseconds taken by all reduce tasks=503049216
	Map-Reduce Framework
		Map input records=3334
		Map output records=22022
		Map output bytes=212979
		Map output materialized bytes=86648
		Input split bytes=1555
		Combine input records=22022
		Combine output records=6304
		Reduce input groups=5956
		Reduce shuffle bytes=86648
		Reduce input records=6304
		Reduce output records=5956
		Spilled Records=12608
		Shuffled Maps =16
		Failed Shuffles=0
		Merged Map outputs=16
		GC time elapsed (ms)=982150
		CPU time spent (ms)=105760
		Physical memory (bytes) snapshot=1453776896
		Virtual memory (bytes) snapshot=32936804352
		Total committed heap usage (bytes)=1927880704
	Shuffle Errors
		BAD_ID=0
		CONNECTION=0
		IO_ERROR=0
		WRONG_LENGTH=0
		WRONG_MAP=0
		WRONG_REDUCE=0
	File Input Format Counters 
		Bytes Read=126112
	File Output Format Counters 
		Bytes Written=59112
23/06/17 10:40:00 INFO mapred.ClientServiceDelegate: Application state is completed. FinalApplicationStatus=SUCCEEDED. Redirecting to job history server
Exception in thread "main" java.io.IOException: Job status not available 
	at org.apache.hadoop.mapreduce.Job.updateStatus(Job.java:331)
	at org.apache.hadoop.mapreduce.Job.isSuccessful(Job.java:616)
	at org.apache.hadoop.mapreduce.Job.waitForCompletion(Job.java:1323)
	at wordcount.WordCount.main(WordCount.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.hadoop.util.RunJar.run(RunJar.java:221)
	at org.apache.hadoop.util.RunJar.main(RunJar.java:136)
